.snippets/code/build/fast-transfers/how-to-solver/auction-1.js

```diff
@@ -0,0 +1,33 @@
+import { Connection, Keypair } from "@solana/web3.js";
+import * as wormholeSdk from "@certusone/wormhole-sdk";
+
+const YOUR_RPC_URL = 'INSERT_RPC_URL';
+const MATCHING_ENGINE_PROGRAM_ID = 'INSERT_MATCHING_ENGINE_PROGRAM_ID';
+const USDC_MINT_ADDRESS = 'INSERT_USDC_MINT_ADDRESS';
```

Comment:
```suggestion
const rpcUrl = 'INSERT_RPC_URL';
const matchingEngineProgramId = 'INSERT_MATCHING_ENGINE_PROGRAM_ID';
const usdcMintAddress = 'INSERT_USDC_MINT_ADDRESS';
```

No need to put the variables into use all uppercase letters, unless they are exported constants. So here they are fine as camel case